### PR TITLE
[Student][E2E][MBL-13000,MBL-13004]: E2E tests for pages & discussions

### PR DESCRIPTION
**Pull request by joehoag**
Note that:
--Discussions includes announcements
--Some a11y issues were also fixed

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

File: apps/student/src/main/res/layout/fragment_create_discussion.xml [hunk 1]
```diff
@@ -58,11 +58,17 @@
...
                 android:textColorHint="@color/defaultTextGray"
                 app:hintTextAppearance="@style/TextInputLabel">
 
+                <TextView
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/title"
+                    android:labelFor="@+id/editDiscussionName"
+                    android:visibility="gone"/>
+
```
Comment (joehoag) [inline apps/student/src/main/res/layout/fragment_create_discussion.xml]: Arggh... This was the only way that I could get the a11y "No speakable text" issue for the control below to clear.

File: apps/student/src/main/res/layout/fragment_discussions_details.xml [hunk 2]
```diff
@@ -306,13 +306,16 @@
...
+                        android:minHeight="48dp"
                         android:layout_marginTop="4dp" />
 
                     <TextView
                         android:id="@+id/replyToDiscussionTopic"
                         style="@style/TextFont.Regular"
                         android:layout_width="wrap_content"
                         android:layout_height="wrap_content"
+                        android:minHeight="48dp"
+                        android:minWidth="48dp"
```
Comment (joehoag) [inline apps/student/src/main/res/layout/fragment_discussions_details.xml]: A11y fixes.

File: libs/rceditor/build.gradle [hunk 3]
```diff
@@ -92,5 +92,5 @@ dependencies {
     implementation Libs.ANDROIDX_FRAGMENT
     implementation Libs.ANDROIDX_VECTOR
 
-    implementation 'jp.wasabeef:richeditor-android:1.2.2'
+    api 'jp.wasabeef:richeditor-android:1.2.2'
```
Comment (joehoag) [inline libs/rceditor/build.gradle]: I had to make this change in order to be able to access RCETextEditors from our UI tests.

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1005496&projectid=20)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: apps/student/src/main/res/layout/fragment_create_discussion.xml (hunk 1 above)
Comment (JordanMarshall) [reply apps/student/src/main/res/layout/fragment_create_discussion.xml]: I think this should be fine, maybe just throw in a small comment in the XML so future devs know what they're looking at

File: libs/rceditor/build.gradle (hunk 3 above)
Comment (JordanMarshall) [reply libs/rceditor/build.gradle]: 👍

File: apps/student/src/main/res/layout/fragment_discussions_details.xml (hunk 2 above)
Comment (JordanMarshall) [reply apps/student/src/main/res/layout/fragment_discussions_details.xml]: 👍

**Pull request merged**